src/main/java/athena/Ui.java

```diff
@@ -222,4 +223,12 @@ public void printExitMessage() {
     public void printTimetable(Timetable timetable) {
         System.out.println(timetable);
     }
+
+    public void printStorageLoadFail() {
+        System.out.println("Storage loading has failed");
```

Comment:
Could you add a "\n" at the end to make the spacing of the Ui outputs consistent?

Reply:
It actually quits if it hits this part so the user would not notice